src/main/java/org/springframework/integration/dsl/core/PollerSpec.java

```diff
@@ -109,9 +110,21 @@ public PollerSpec advice(Advice... advice) {
 	 * provided {@code PlatformTransactionManager} for the {@code pollingTask}.
 	 * @param transactionManager the {@link PlatformTransactionManager} to use.
 	 * @return the spec.
+	 * @deprecated - since 1.2 use {@link #transactional(TransactionInterceptor)}
 	 */
+	@Deprecated
```

Comment:
OK; makes sense; but I guess we need to explain in the javadocs it just gets default props.